### DISCUSSION TITLE
About the default blueprint...

### DISCUSSION THREAD

**Discussion by cdillon** [statamic/cms#8625] [General]
1. I'm learning Statamic.
2. I'm really good at breaking things.

I experienced a problem when following the Quick Start Guide.

Upon saving a blueprint for the first time, either unmodified or after the first customization (like when adding the Featured Image to the Blog blueprint), the default blueprint is deleted. So the yet-unmodified Page blueprint loses the default Author and Template fields.

Restoring the `default.yaml` file fixes it, as does manually creating a blueprint file from `default.yaml`, but any subsequent customizations will delete the default again.

Is that the intended behavior?

If so, maybe the Quick Start Guide should recommend saving the Page blueprint as is before creating the Blog collection.

**Comment by santossoares** [comment]
I got this behavior too and took some time to figure out what's going on.

There is no Page blueprint, it uses the default.yaml file, so after creating and modifiying the a new blueprint, the default.yaml gets deleted and the Page loses the author and template that it gets from the default.yaml since there is no Page blueprint.
I think this is a bug.

Is that supposed to happen?

**Comment by cdillon** [comment]
I think deleting the default makes more sense for Statamic's "start simple and add things as needed" approach. I prefer that because not every content type will need the same fields (especially Author). It just confused me when the fields disappeared and the guide made no mention of the default blueprint. But I did enjoy digging into the code to figure it out so I will probably not forget this.

**Comment by jasonvarga** [comment]
Yeah we should probably just remove the `default.yaml` blueprint.
